src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/SwaggerRoutes.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsde.workbench.leonardo.api
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.Directives._
+import akka.stream.scaladsl.Flow
+import akka.util.ByteString
+import org.broadinstitute.dsde.workbench.leonardo.config.SwaggerConfig
+
+trait SwaggerRoutes {
+  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/2.2.5"
+
+  val swaggerConfig: SwaggerConfig
+
+  val swaggerRoutes: server.Route = {
+    path("") {
+      get {
+        parameter("url") { urlparam =>
```

Comment:
It looks like this redirects to the same URI but with the query string stripped out? What is the `url` parameter -- is it a Swagger thing that we don't care about?

Since `urlparam` is not used I'd recommend doing:
```
parameter("url") { _ =>
  ...
}
```